### DISCUSSION TITLE
AD5M Pro Specific Question (For the high-speed Klipperers among us!)

### DISCUSSION THREAD

**Discussion by caall99** [xblax/flashforge_ad5m_klipper_mod#98] [General]
Hey all,

Been really enjoying this mod! Thank you for your efforts! I am very impressed with the AD5M Pro printer - especially after installing the Klipper-mod. In addition, I opened up the back of the printer and disassembled the external fan shroud, fixed the servo arm (and config file), and printed and installed a stand-off for the PCB fan. All of that helped with the noise. Unfortunately, I am noticing a rattling noise from the area behind the LCD. It's a rather loud and unhealthy sound. X & Y Accelerations are at <10k mm/s^2 and SQV is at 25mm/s. X & Y Speeds <300mm/s. Anyone else experiencing this? I am asking here, as I know this group is pushing this printer hard like I am... and would likely experience similar QC issues etc... as i am. Thanks!

**Comment by xblax** [comment]
Don't have the Pro, but I would in general recommend to limit Max Accel to Decel to 2500. Maybe that also helps with the rattling noise.
`SET_VELOCITY_LIMIT ACCEL_TO_DECEL=2500`

**Reply by caall99** [reply]
Appreciate the response and suggestion, but the printer is capable of much more accel to decel, just need to pin down source of noise. I ran it without accel to decel at 20k accel, and it printed a fine benchy with no skipped steps. Just curious if anyone is hearing a similar noise. Thanks

**Comment by consp** [comment]
> I am noticing a rattling noise from the area behind the LCD

I now have a pro and do not have that issue. Can you describe it a bit more or record it on a video and post that? Is it periodic with the extruder (e.g. as the extruder moves it makes a sound) or is it continous clicking noise? The first one could be a lose wire/the ribbon cable klapping agains something when there are vibrations in the system due to the extruder moving, taping down the wires of the camera and display should fix that. Seconds one could be a lot of things.
